.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,37 @@
+name: Docker image CD
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
```

Comment:
```
# platforms: linux/amd64,linux/arm64
platforms: linux/arm64
```

Reply:
I'm still not sure it's possible to build one architecture using Travis and another using Github Actions. I'd imagine you'd end up with only one of the architectures being saved under the `latest` tag instead of both.

Reply:
Yes, I also think it is not possible with _Travis_ and _GitHub Actions_ to have the `latest` tag for both systems. But I guess the `arm64` community would be still happy to have a build at all. And after a few weeks, when the workflow is proven, we can switch both builds to _GitHub Actions_ (including the `latest` tag).

Reply:
I've set it up to now only build an image for the ``arm64`` architecture, and it will also only tag it using the ``arm64`` tag.